profile-management/OktaProfileManagement/UserProfileController.swift

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2019 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import OktaOidc
+
+class UserProfileController: UIViewController {
+    private var profileManager: ProfileManager?
+    private var currentUser: User? {
+        return profileManager?.user
+    }
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var subtitleLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var mobilePhoneLabel: UILabel!
+    @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet weak var createdLabel: UILabel!
+    @IBOutlet weak var activatedLabel: UILabel!
+    @IBOutlet weak var statusChangedLabel: UILabel!
+    @IBOutlet weak var lastLoginLabel: UILabel!
+    @IBOutlet weak var lastUpdateLabel: UILabel!
+    @IBOutlet weak var passwordChangedLabel: UILabel!
+    
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
Not clear why you use `viewWillAppear` for the initial data download. I believe you don't need re-request data on each `viewWillAppear` callback. `viewDidLoad` should be sufficient for that case